### DISCUSSION TITLE
初学者想做一个类似这个小机器人的上位机数字孪生模型

### DISCUSSION THREAD

**Discussion by TriAu** [peng-zhihui/ElectronBot#199] [Q&A]
想做一个塔吊的数字孪生模型

想了解一下建立类似的数字孪生模型需要哪些方面的基础知识，需要用到哪些工具？

另外像稚晖君这个，选用unity开发上位机是有哪方面的考虑？是否普适？

请求各位大佬指点！

**Comment by Eilgnaw** [comment]
我个人觉得用 unity 就是为了这个数字孪生.

**Reply by TriAu** [reply]
感谢指点
